src/Models/BaseElement.php

```diff
@@ -1140,18 +1143,14 @@ public function getIcon()
     /**
      * Get a description for this content element, if available
      *
-     * @deprecated 5.3.0 Use the description configuration property and localisation API directly instead.
+     * @deprecated 5.3.0 Use i18n_classDescription() instead.
      *
      * @return string
      */
     public function getDescription()
     {
-        Deprecation::notice('5.3.0', 'Use getTypeNice() or the description configuration property directly instead.');
-        $description = $this->config()->uninherited('description');
-        if ($description) {
-            return _t(__CLASS__ . '.Description', $description);
-        }
-        return '';
+        Deprecation::notice('5.3.0', 'Use i18n_classDescription() instead.');
```

Comment:
Was deprecated in 5.3.0 so keeping that version there, though the advice is now different.